scripts/dev

```diff
@@ -342,14 +342,33 @@ namespace :db do
   end
 
   desc "Dump DB to file"
-  task :dump do
-    sh "pg_dump -T *.*flyway* -T *.*existing_model --data-only postgres > scripts/data/local-db.sql"
+  task :dump, [:file] do |t, args|
+    file = args.to_a[0]
+
+    if file == nil
+      dump()
+    else
+      if file.start_with?("/")
+      else
+      file = "scripts/data/"+file+".sql"
+      end
+     dump(file)
+    end
   end
 
   desc "Load development dataset from file"
-  task :seed do
-    # Seed DB
-    `psql < scripts/data/local-db.sql > /dev/null`
+  task :seed, [:file] do |t, args|
+    file = args.to_a[0]
+
+    if file == nil
+      seed()    
+    else
+      if file.start_with?("/")
+      else
```

Comment:
What does this `if` with an immediate `else` do?

Also, I assume the `file = ` line falls under the `else`? (If so, indentation might need a touch up to make it clear)

Reply:
Good call about the indentation! I added it. 

The if immediate else was meant to ignore the situation where someone passes an explicit path to the function (something like a temp demo file you keep on your desktop, but isn't needed for the repo eg `/Users/cloak/Desktop/local-db.sql`) .

 But looking at it, I think it is a poor way to structure that logic.  I'm changing it to this, which I think is clearer
 
 ```ruby
 
      if !file.start_with?("/") #files that start with / are fully qualified path names
           file = "scripts/data/"+file+".sql"
      end
 ```